docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -127,6 +127,42 @@ supports the following features:
   :ref:`iceberg-materialized-views`
 * :ref:`sql-views-management`
 
+.. _iceberg-alter-table-execute-optimize:
+
+ALTER TABLE EXECUTE OPTIMIZE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector offers :ref:`ALTER TABLE EXECUTE <alter-table-execute>`
+``OPTIMIZE`` command for collapsing files which correspond to the
+current snapshot of a table into fewer larger files.
+In case that the table is partitioned, the collapsing will take place
+for each partition taken into consideration for optimization.
+This operation improves read performance and reduces disk usage.
+
+All files with a size below the optional ``file_size_threshold``
+parameter (default value for the threshold is ``100MB``) are
+collapsed:
```

Comment:
i know we used "collapse" in hive docs, but i am not convinced this is the best verb. can we improve?

also, we shouldn't give impression that we guarantee "collapsing" into one file.

Reply:
changed to `merge`